components/samsung_ac/protocol_nasa.cpp

```diff
@@ -699,6 +699,13 @@ namespace esphome
                     ESP_LOGW(TAG, "s:%s d:%s NASA_OUTDOOR_CONTROL_WATTMETER_TOTAL_SUM_ACCUM %f", source.c_str(), dest.c_str(), value);
                     return;
                 }
+                if ((uint16_t)message.messageNumber == 0x4235)

```

Comment:
Can you please extend the enum with VAR_IN_TEMP_WATER_HEATER_TARGET_F and use that here?